tests/fsharp/typecheck/sigs/neg106.bsl

```diff
@@ -46,16 +46,16 @@ neg106.fs(40,18,40,32): typecheck error FS0041: No overloads match for method 'M
 Known types of arguments: string * inref<int>
 
 Available overloads:
- - static member C.M: a: int * x: byref<int> -> unit // Argument 'a' doesn't match
- - static member C.M: a: string * x: byref<int> -> unit // Argument 'x' doesn't match
+ - static member C.M: a: int * [<System.Runtime.InteropServices.Out>] x: byref<int> -> unit // Argument 'a' doesn't match
+ - static member C.M: a: string * [<System.Runtime.InteropServices.Out>] x: byref<int> -> unit // Argument 'x' doesn't match
```

Comment:
I'm not 100% sure we want to show attributes in the help message.
For sure not `[<In>]`/`[<Out>]` for byrefs, those should probably be turned into `inref`/`outref`.

Reply:
This attribute is literally present in the source code:
https://github.com/dotnet/fsharp/blob/4a052ae0713d69b21d8b9f02e2e360436f2b83da/tests/fsharp/typecheck/sigs/neg106.fs#L28
So I don't think it is a stretch here that it is included.

Reply:
Ah, got it. Thought that it's implicit.